fastlane/amplify_pods.rb

```diff
@@ -40,11 +40,22 @@ module AmplifyPods
       plist_paths: [
         "AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Resources/Info.plist",
         "AmplifyPlugins/API/AWSAPICategoryPlugin/Info.plist",
-        "AmplifyPlugins/Storage/AWSS3StoragePlugin/Resources/Info.plist"
+        "AmplifyPlugins/Storage/AWSS3StoragePlugin/Resources/Info.plist",
+        "AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Resources/Info.plist"
```

Comment:
1. What about `AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Info.plist`?
1. nit: Alphabetize to make it easier to see what's missing